hw16.js

```diff
@@ -53,28 +72,80 @@ function addProductToList(product){
     selectBox.add(option);
 }
 
+function onCangeReset(){
+    document.getElementById('kg').value = "";
+    document.getElementById('count').value = "";
+}
 
 function onChangeProduct(products) {
     var idProduct = selectBox.options[selectBox.selectedIndex].id;
     var item = products.find(item => item.id == idProduct);
     return item
+};
+
+let sum = [];
+
+function getSum(item) {
+    let count = document.getElementById('count').value;
+    sum.push(item.toBuySum(count));
```

Comment:
better will be if function will return some value instead of pushing into global var